ouroboros-consensus-diffusion/changelog.d/20250304_135940_jasataco_release_10_3.md

```diff
@@ -1,28 +0,0 @@
-<!--
-A new scriv changelog fragment.
-
-Uncomment the section that is right (remove the HTML comment wrapper).
--->
-
-<!--
-### Patch
-
-- A bullet item for the Patch category.
-
--->
-<!--
-### Non-Breaking
-
-- A bullet item for the Non-Breaking category.
-
--->
-### Breaking
-
-- Update to latest `ouroboros-network` release:
-    | Package                     | Version |
-    |-----------------------------|---------|
-    | network-mux                 | 0.7     |
-    | ouroboros-network           | 0.20    |
```

Comment:
I also updated this line to say 0.20.1 following the fragment I added in the other PR, instead of having two separate lines in the changelog for consecutive updates.